kitsune/customercare/forms.py

```diff
@@ -10,21 +11,34 @@ class ZendeskForm(forms.Form):
     """Form for submitting a ticket to Zendesk."""
 
     product = forms.CharField(disabled=True, widget=forms.HiddenInput)
-    category = forms.ChoiceField(
-        label=_lazy("What do you need help with?"), choices=CATEGORY_CHOICES
+    email = forms.EmailField(
+        label=_lazy("Email"), required=False, disabled=True, widget=forms.HiddenInput
```

Comment:
Since this field can be used in both states (loginless and loginfull) we can remove `required` and `disabled` flags. Or remove `disabled` and keep `required=True`

Reply:
Pending a meeting....